docs/get_started.ipynb

```diff
@@ -291,14 +287,37 @@
         "Additional TensorBoard plugins are automatically enabled when you log other types of data. For example, the Keras TensorBoard callback lets you log images and embeddings as well. You can see what other plugins are available in TensorBoard by clicking on the \"inactive\" dropdown towards the top right."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vsowjhkBdkbK"
+      },
+      "source": [
+        "## Hosting your TensorBoard to share with others\n",
```

Comment:
I think this section should be at the end of the tutorial. We first teach the two ways to use TensorBoard, then share how it can be hosted.

Reply:
I'd suggest including TensorBoard.dev in the header. Perhaps "TensorBoard.dev: Host and share your ML experiment results"?